examples/quickstart/20-pod.yaml

```diff
@@ -55,9 +53,6 @@ spec:
         name: sonobuoy-plugins-cm
     # NOTE: This will drop the results on the hostpath of the machine as an example.
     # We recommend using a persistent volume for more permanent recording.
-    - name: output-volume
-      hostPath:
```

Comment:
Sure, that's probably a decent idea, even though it's roughly equivalent to no volume at all, since /tmp is writable inside the container.

With emptyDir it's easier to change the YAML to a different type of persistent volume for your purposes, without having to remember what path to use, etc... I'll change it to emptyDir.